### DISCUSSION TITLE
Route import formats?

### DISCUSSION THREAD

**Discussion by Cer-o** [devemux86/cruiser#213] [Q&A] [answered]
Hello everyone. I am designing the routes for this summer's trip.
I do the design on Kurviger web, including waypoints and their names.
When I import it in Cruiser in .kurviger format, I see that it only imports the start and the end, but the path is totally different. However, if I import it in .gpx format, I see that the import is correct, the same route, waypoints and their names.
I thought that Cruiser was compatible with the .kurviger format, but I see that it is not.

**Comment by DolomitesTom** [comment]
It's working fine here. The format is compatible, not the routing engine. Either you have to to switch off settings - navigation - route calculation or put more shaping points in kurviger web. I guess you didn't imported as overlay? Can you send the kurviger web link?

**Comment by devemux86** [comment]
@Cer-o 

_Cruiser is compatible with the .kurviger format!_

- Cruiser imports Kurviger routes offline and exactly as they are **without changes**.

- Importing gpx files requires route calculation during import.
(unless we import navigation gpx files)

In navigation rerouting, Cruiser only calculates the route to the next waypoint, so your **curved routes remain unchanged**.

Can you describe in more detail what steps you take so that we can help with your import process?

Please see below how to import Kurviger routes:

<img src="https://github.com/devemux86/cruiser/assets/3484020/7337b894-3c2f-40c3-a715-92a39226d66d" width="50%"/>

**Comment by WolfgangZ42** [comment]
> I thought that Cruiser was compatible with the .kurviger format, but I see that it is not.

Believe me - it is compatible! I ride all my .kurviger routes with Cruiser - no problem at all.

**Comment by Cer-o** [comment]
Hello everyone and thanks for your interest.
I have planned the route in Kurviger with 86 waypoints (image nº1) and exported in kurviger format.
When I import it in Cruiser, it creates a different route without waypoints (image nº2).
When I import it in Cruiser in .gpx format, it reproduces the original route with the waypoints (image nº3)

![image](https://github.com/devemux86/cruiser/assets/122309655/f0ef69d9-a01a-4b68-b9b8-491d854e6ec8)
![image](https://github.com/devemux86/cruiser/assets/122309655/537e60e5-b072-42cf-9a22-d2483d5b4495)
![image](https://github.com/devemux86/cruiser/assets/122309655/ce003a4f-5d6f-4082-8b90-60191197e47a)

**Reply by devemux86** [reply]
@Cer-o  can you post the .kurviger file so we can make tests?

What exact options do you use each time in the import dialog?
It is not possible to import a route format as a different format.

Please see the image above: https://github.com/devemux86/cruiser/discussions/213#discussioncomment-6170223
It shows how the import dialog should be to import kurviger routes.

**Comment by Cer-o** [comment, marked as answer]
Apparently I was doing something wrong when importing, because now I have tried it several times and it has worked.
I have transferred the .kurviger file to Telegram and after "clicking" it I have chosen to open it with Cruiser... and it has worked perfectly.

The truth is that I must be clumsier than I imagined.
I apologize for the inconvenience caused.

**Reply by devemux86** [reply]
@Cer-o no problem, we are all here to help!

Please check carefully the selected options in the import dialog before proceeding, as they affect the import process.
